lib/Sapi.php

```diff
@@ -71,6 +71,11 @@ static function sendResponse(ResponseInterface $response) {
         $body = $response->getBody();
         if (is_null($body)) return;
 
+        if (is_callable($body)) {
+            $body();
```

Comment:
I'm with @petrkotek on this. `generateMultiStatus` really should move out of `Sabre\DAV\Server` anyway. It's a leftover from the extremely early days.

Usually I think it's reasonable to assume that the thing that sets the body, also has the ability to access the response object.